newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -358,7 +403,6 @@ export default class PixiResourcesLoader {
     if (resource.getKind() !== 'video') return invalidTexture;
 
     const url = ResourcesLoader.getResourceFullUrl(project, resourceName, {
-      disableCacheBurst: true, // Disable cache bursting for video because it prevents the video to be recognized as such (for a local file)
```

Comment:
Why is this problem commented not appearing any more?

Reply:
It was introduced in 2019 (https://github.com/4ian/GDevelop/commit/db93a25b331c207d2572744602a7034268e3571d) so I think that it maybe concerned the other rendering engine, or Pixi evolved since then. I did not dig into it